lib/posix/posix_openbsd_amd64.nim

```diff
@@ -313,6 +315,12 @@ type
     si_band*: int      ## Band event for SIGPOLL.
     si_value*: SigVal  ## Signal value.
 
+template sa_sigaction*(v: Sigaction): proc (x: cint, y: ptr SigInfo, z: pointer) {.noconv.} =
```

Comment:
Why a public template for an importc'd C function ?, can it be just a `proc` ?. 🤔

Reply:
sa_sigaction and sa_handler are within an union.

Reply:
Though in fact when I rewrote them, I didn't realize the previous also works.